abby's algorithms/modeanswer.js

```diff
@@ -0,0 +1,29 @@
+function SimpleMode(arr) {
+
+    var map = {};
+    var mode = 0;
+    var highestFreq = 0;
+
+    for (i = 0; i < arr.length; i++) {
```

Comment:
generally best practice in Javascript to declare `var i` at some point, either in the for loop or up at the top with your other variables